src/compressed_tensors/quantization/quant_args.py

```diff
@@ -225,7 +225,7 @@ def validate_model_after(model: "QuantizationArgs") -> Dict[str, Any]:
 
         elif observer is None:
             # default to minmax for non-dynamic cases
```

Comment:
we'll want to change this comment as well
```suggestion
            # default to mse for non-dynamic cases
```

Reply:
Good catch!